From 0553455813a1f717ef50cc911e4db73da3d23c32 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 22 Mar 2017 18:47:40 -0700
Subject: [PATCH 1/2] Enable system malloc on all linux

Dont trigger on being just android or glibc
internal malloc falls flat on musl based systems
as well. So switch to using system malloc

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 config_f.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config_f.h b/config_f.h
index fb9e69b..c207c25 100644
--- a/config_f.h
+++ b/config_f.h
@@ -147,7 +147,7 @@
  *		This can be much slower and no memory statistics will be
  *		provided.
  */
-#if defined(__MACHTEN__) || defined(PURIFY) || defined(MALLOC_TRACE) || defined(_OSD_POSIX) || defined(__MVS__) || defined (__CYGWIN__) || defined(__GLIBC__) || defined(__OpenBSD__) || defined(__APPLE__) || defined (__ANDROID__) || defined(__NetBSD__) || !defined(HAVE_WORKING_SBRK)
+#if defined(__MACHTEN__) || defined(PURIFY) || defined(MALLOC_TRACE) || defined(_OSD_POSIX) || defined(__MVS__) || defined (__CYGWIN__) || defined(__linux__) || defined(__OpenBSD__) || defined(__APPLE__) || defined (__ANDROID__) || defined(__NetBSD__) || !defined(HAVE_WORKING_SBRK)
 # define SYSMALLOC
 #else
 # undef SYSMALLOC
-- 
2.34.1

